app/api/models/dto/user_in.go

```diff
@@ -13,3 +13,13 @@ type UserSignIn struct {
 	Email       string `json:"email"`
 	RawPassword string `json:"password"`
 }
+
+type UserUpdate struct {
+	Email       string `json:"email"`
+	PhoneNumber string `json:"phone_number"`
+	RawPassword string `json:"password"`
```

Comment:
Are there any secondary considerations here? If you somehow get somebody's session, you probably shouldn't be able to click 2 buttons and change their password. This should be locked behind a check that the user actually knows their current password.

I.E. there should be a separate endpoint for updating RawPassword.